src/bridge/cockpitcpusamples.c

```diff
@@ -188,12 +205,26 @@ cockpit_cpu_temperature (CockpitSamples *samples)
       g_strchomp (name);
 
       // compare device name with CPU names
-      // Intel: coretemp, AMD: k8temp or k10temp, ARM: cpu_thermal
+      // Intel: coretemp, AMD: k8temp or k10temp, ARM: cpu_thermal, Asus motherboard: atk0110
       if (g_str_equal (name, "coretemp") || g_str_equal (name, "cpu_thermal") ||
-          g_str_equal (name, "k8temp") || g_str_equal (name, "k10temp"))
+          g_str_equal (name, "k8temp") || g_str_equal (name, "k10temp") || g_str_equal (name, "atk0110"))
         {
           // hwmon contains CPU info
-          sample_cpu_sensors (samples, i);
+          detect_cpu_sensors (devices, name, i);
         }
     }
 }
+
+void
+cockpit_cpu_temperature (CockpitSamples *samples)
+{
+  static GList *devices = NULL;
+  static gboolean initialized = FALSE;
+  if (initialized == FALSE)
```

Comment:
These are booleans, so `if (!initialized)`. If nothing else requires changes any more, I'll let that slide, though.